.changeset/changelog-config.cjs

```diff
@@ -0,0 +1,53 @@
+// @ts-check
+
+/** @type import('@changesets/types').GetReleaseLine */
+async function getReleaseLine(changeset) {
+  let packages = changeset.releases.map(release => release.name);
+
+  if (packages.length >= 2) {
+    packages = packages.filter(packageName => packageName !== 'prosekit');
```

Comment:
@Saul-Mirone In ProseKit, I want to publish multiple packages but have one CHANGELOG.md file that contains updates from all packages. That's why I write a this custom `changelog-config.cjs`. You likely want to use the default Changeset changelog config (by removing `changelog-config.cjs` and also related content in `.changeset/config.json`), or update this script to match blocksuite's requirement.